resources/solr-configuration/src/main/resources/patients/conf/schema.xml

```diff
@@ -61,6 +61,7 @@
     <field name="reporter" type="string" indexed="false" stored="true" required="true" />
     <field name="visibility" type="string" indexed="true" stored="true" required="false" />
     <field name="accessLevel" type="int" indexed="true" stored="true" required="false" />
+    <field name="phenotype_ancestors" type="string" indexed="true" stored="true" multiValued="true"/>
```

Comment:
Also, we do not index any of extended fields : extended_phenotype or extended_prenatal_phenotype or extended_negative_phenotype. So which of them is would make sence to index to run solr query in patient network? Additionally, I discovered that by some reason patient-extended-phenotype-updater is not updating negative_prenatal_phenotype to its extended version with others, but we do have this field in the PatientClass. Does it mean we shouldn't consider it?